### DISCUSSION TITLE
Updating MVT

### DISCUSSION THREAD

**Discussion by atvnuk** [mvt-project/mvt#261] [Q&A] [answered]
Hi I just wanted to ask what is the best way of upgrading / updating the MVT toolkit 
 MVT - Mobile Verification Toolkit
                https://mvt.re
                Version: 1.5.3
                Version 1.5.4 is available! Upgrade mvt!

**Comment by botherder** [comment, marked as answer]
`pip3 install -U mvt`

**Reply by atvnuk** [reply]
Thank you!
